apps/web/package.json

```diff
@@ -106,6 +106,7 @@
     "qrcode": "^1.5.1",
     "raw-body": "^2.5.1",
     "react": "^18.2.0",
+    "react-beautiful-dnd": "^13.1.1",
```

Comment:
are you using this package anywhere?

Reply:
No I am trying to make it work using framer-motions Reorder, this is still a WIP will remove it soon. Making it draft till then